src/libcore/ptr.rs

```diff
@@ -2790,7 +2790,7 @@ impl<T: ?Sized> Unique<T> {
     }
 
     /// Acquires the underlying `*mut` pointer.
-    pub fn as_ptr(self) -> *mut T {
+    pub const fn as_ptr(self) -> *mut T {
```

Comment:
cc @Centril is it too dangerous that just adding a `const` doesn't require anything like a `#[const_stable(since = "1.2.3")]` attribute?

Reply:
Dangerous from what perspective? If it's used in a min-const-fn it will also need to be min-const-fn... or do you mean from the POV of accidental stabilization?

Reply:
yes, accidental stabilization. Like making something stable that is already `const fn` stabilizes the constness along with it.

Reply:
Do you mean any stabilized const fn should have something like a `#[const_stable]` attribute?

Reply:
Ah... idk if I find it risky rising to the level of dangerous but there's some risk, no doubt. @varkor wanted it from a different perspective (https://github.com/rust-lang/rust/issues/57562#issuecomment-453794072) but I thought it too much of an annotation burden. However, from the POV of accidental stabilization, `#[const_stable(...)]` makes sense to me.

Can you open up an issue and cc me and varkor?

Reply:
@TimDiekmann Yeah, that's the idea.